src/loop_libraries/library_collier.hpp

```diff
@@ -0,0 +1,47 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#ifndef LOOP_LIBRARY_COLLIER
+#define LOOP_LIBRARY_COLLIER
+
+#include "loop_library_interface.hpp"
+
+#define REDEFINE(R,ARGS,NAME) std::complex<double> NAME ARGS noexcept;
+
+namespace looplibrary
+{
+class Collier : public Loop_library_interface
+{
+   private:
+      double current_mu2_uv;
+      void initialize() noexcept;
+      void set_mu2_uv(double) noexcept;
+   public:
+      Collier() : current_mu2_uv(1.0) { initialize(); }
+      BOOST_PP_SEQ_FOR_EACH(REDEFINE,(A_ARGS),A_SEQ)
+      BOOST_PP_SEQ_FOR_EACH(REDEFINE,(B_ARGS),B_SEQ)
+      BOOST_PP_SEQ_FOR_EACH(REDEFINE,(C_ARGS),C_SEQ)
+      BOOST_PP_SEQ_FOR_EACH(REDEFINE,(D_ARGS),D_SEQ)
+      void A(std::complex<double> (&)[1], A_ARGS) noexcept;
```

Comment:
Declare this and the following functions as `override`.
(This applies to all classes that inherit from `Loop_library_interface`.)

Reply:
Done in 121b547.